jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -196,33 +199,34 @@ public boolean clearCurrentSelection( CellPreviewEvent<ProcessInstanceSummary> e
 
                     @Override
                     public DefaultSelectionEventManager.SelectAction translateSelectionEvent( CellPreviewEvent<ProcessInstanceSummary> event ) {
+                        DefaultSelectionEventManager.SelectAction ret = DefaultSelectionEventManager.SelectAction.DEFAULT;
                         NativeEvent nativeEvent = event.getNativeEvent();
                         if ( BrowserEvents.CLICK.equals( nativeEvent.getType() ) ) {
                             // Ignore if the event didn't occur in the correct column.
-                            if ( extendedPagedTable.getColumnIndex( actionsColumn ) == event.getColumn() ) {
```

Comment:
@rbarriuso your fix seems reasonable to me, my only suggestion would be to handle it on a separated JIRA and PR to target this problem specifically across all list tables, wdyt?